src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return cost - 50;
+  } else if (days >= 3) {
+    return cost - 20;
```

Comment:
Use a constant to represent the short-term rental threshold and the associated discount, instead of hardcoding the values '3' and '20'.

Reply:
Still not fixed